red.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 /**
- * Copyright 2013, 2016 IBM Corp.
+ * Copyright 2013, 2016 IBM Corp., 2016 Lyteworx LLC.
```

Comment:
as above CLA comment, a dual copyright will be even more complex to try and maintain